ChatQnA/kubernetes/helm/README.md

```diff
@@ -28,3 +28,224 @@ helm install chatqna oci://ghcr.io/opea-project/charts/chatqna  --set global.HUG
 ```
 
 See other *-values.yaml files in this directory for more reference.
+
+## Deploy on AMD ROCm using Helm charts from the binary Helm repository
+
+```bash
+mkdir ~/chatqna-k8s-install && cd ~/chatqna-k8s-install
+```
+
+### Cloning repos
+
+```bash
+git clone git clone https://github.com/opea-project/GenAIExamples.git
+```
+
+### Go to the installation directory
+
+```bash
+cd GenAIExamples/ChatQnA/kubernetes/helm
+```
+
+### Settings system variables
+
+```bash
+export HFTOKEN="your_huggingface_token"
+export MODELDIR="/mnt/opea-models"
+export MODELNAME="meta-llama/Meta-Llama-3-8B-Instruct"
+```
+
+### Setting variables in Values files
+
+#### If ROCm vLLM used
+```bash
+nano ~/chatqna-k8s-install/GenAIExamples/ChatQnA/kubernetes/helm/rocm-values.yaml
```

Comment:
Could assume you are already in the correct directory, namely
~/chatqna-k8s-install/GenAIExamples/ChatQnA/kubernetes/helm